### PR TITLE
Travis testing

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

**Comment by isuftin** [comment]
@zmoore-usgs Let me know what you think

File: configuration/local/local.jmeter.properties [hunk 1]
```diff
@@ -1,9 +1,9 @@
-WATERAUTH_HOST=localhost
-SERVER_HOST=localhost
-GATEWAY_HOST=localhost
-LEGACY_CRU_HOST=localhost
-DDOT_HOST=localhost
-TRANSFORMER_HOST=localhost
+WATERAUTH_HOST=192.168.99.100
```
Comment (zmoore-usgs) [inline configuration/local/local.jmeter.properties]: Should these go back to localhost?

File: configuration/local/config/common/config.env [hunk 2]
```diff
@@ -1 +1 @@
-oauthClientAuthorizationUri=https://water.auth.server:8443/auth/oauth/authorize
+oauthClientAuthorizationUri=https://192.168.99.100:8443/auth/oauth/authorize
```
Comment (zmoore-usgs) [inline configuration/local/config/common/config.env]: Should this go back to water.auth.server?

File: read_results.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+TEST_NAME=$1
+OUTPUT_FILE="$(pwd)/tests/output/${TEST_NAME}/jmeter-output/jmeter-testing-${TEST_NAME}.jtl"
+
+if [ ! -f $OUTPUT_FILE ]; then
+  echo "The test results at $OUTPUT_FILE do not exist"
+  exit 1
+fi
+
+cut -d ',' -f8 $OUTPUT_FILE | \
```
Comment (zmoore-usgs) [inline read_results.sh]: Fancy, didn't know about this :+1:

**Comment by isuftin** [comment]
@zmoore-usgs So my updates here do get the tests working but it seems like a bug in the actual code is keeping us from seeing a full success. It's up to you whether or not you'd like to merge this and have it work when upstream code changes fix the issue? 

This PR also gets integration testing working in Travis

https://travis-ci.org/USGS-CIDA/MLR-Integration-Testing/builds/492836727

**Comment by isuftin** [comment]
btw, this PR should be closed and we should be looking @ https://github.com/USGS-CIDA/MLR-Integration-Testing/pull/23

**Pull request closed without merging**